### ISSUE TITLE
What parameters should I set for getting the same result as MATLAB?

### ISSUE THREAD

**Issue by ffbear1993** [laszukdawid/PyEMD#63]
Hi, laszukdawid, I want to use your code to get the same imfs as in MATLAB. I tried the EMD_matlab.py and EMD.py, but it can't. I use the default parameter in Matlab, let's say `emd(x)`, so I want to know how to set the parameters to get the same result.

**Comment by laszukdawid** [comment]
Hi @ffbear1993, unfortunately, it's been a while since I tested PyEMD against Matlab version. I'm not sure what's needed to obtain the same algorithm. EMD_matlab.py was tweaked to obtain the same results but, given that it also depends on external packages and Matlab as internal optimization, this didn't last long. What's also annoying is that EMD is sensitive to small differences in data. A large number of interactions with differing two similar signals largely emphasise the difference and explode. I found eventually that trying to keep Matlab and PyEMD in sync is futile. 

So, I don't know. Sorry.
